From dd07b78f561914ccc6cbe076ae66c380304bf92c Mon Sep 17 00:00:00 2001
From: CN_SZTL <cnsztl@project-openwrt.eu.org>
Date: Wed, 19 Aug 2020 12:42:45 +0000
Subject: [PATCH] build: drop useless deps simd_asm

libjpeg_turbo itself has alreadly set the dependency "simd_asm",
and this will cause cross-compile failed:
```
ERROR Unresolved dependencies.
//:gn_all(//build/toolchain/linux:clang_arm64_openwrt)
  needs //third_party/libjpeg_turbo:simd_asm(//build/toolchain/linux:clang_arm64_openwrt)
```
So, let's drop it in global build file.
---
 src/BUILD.gn | 1 -
 1 file changed, 1 deletion(-)

--- a/src/BUILD.gn
+++ b/src/BUILD.gn
@@ -749,7 +749,6 @@ group("gn_all") {
       if (enable_nacl) {
         deps += [ "//native_client/src/trusted/platform_qualify:vcpuid" ]
       }
-      deps += [ "//third_party/libjpeg_turbo:simd_asm" ]
     }
     if ((is_linux || is_chromeos) && current_toolchain == host_toolchain) {
       deps += [ "//v8:v8_shell" ]
